setup.py

```diff
@@ -47,6 +47,7 @@ def package_files(directory):
         "protobuf>=3.6.0",
         "gitpython>=2.1.0",
         "pyyaml",
+        "pyarrow",
```

Comment:
This should be moved to `extras` - see https://github.com/mlflow/mlflow/pull/3486#discussion_r504420027

Reply:
Ah, missed that - thanks